librocksdb_sys/crocksdb/c.cc

```diff
@@ -4988,6 +4988,18 @@ void ctitandb_decode_blob_index(const char* value, size_t value_size,
   index->blob_size = bi.blob_handle.size;
 }
 
+void ctitandb_encode_blob_index(const ctitandb_blob_index_t& index,
+                                char** value, size_t* value_size) {
```

Comment:
s/value/encoded
s/value_size/encoded_size
`value` may introduce misunderstanding that it is the value of the key.

Reply:
just keep consistent with `ctitandb_decode_blob_index`